tests/test_run_reference_vmm.py

```diff
@@ -2,27 +2,26 @@
 # SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
 """Run the reference VMM and shut it down through a command on the serial."""
 
-import os, signal, subprocess, time
-import pytest
-from subprocess import PIPE, STDOUT
+import os, subprocess, time, fcntl
+from subprocess import PIPE
 import tempfile
 
+import pytest
 
 KERNELS_INITRAMFS = [
     "/tmp/vmlinux_busybox/linux-4.14.176/vmlinux-hello-busybox",
     "/tmp/vmlinux_busybox/linux-4.14.176/bzimage-hello-busybox"
 ]
 
-"""
-Temporarily removed the ("bzimage-focal", "rootfs.ext4") pair from the
-list below, because the init startup sequence in the guest takes too
-long until getting to the cmdline prompt for some reason.
-"""
 KERNELS_DISK = [
     ("/tmp/ubuntu-focal/linux-5.4.81/vmlinux-focal",
      "/tmp/ubuntu-focal-disk/rootfs.ext4"),
+    ("/tmp/ubuntu-focal/linux-5.4.81/bzimage-focal",
+     "/tmp/ubuntu-focal-disk/rootfs.ext4"),
 ]
 
+# No. of seconds after which to give up for the test
+TEST_TIMEOUT = 60
```

Comment:
Isn't this timeout a bit too much?

Reply:
Should I make it 30 seconds,? I will have to rebase the commits once, so I can make it 30 seconds, if that's fine.